tests/api/images/convert_url.test.js

```diff
@@ -0,0 +1,37 @@
+require('should')
+const { authReq } = require('../utils/utils')
+const endpoint = '/api/images?action=convert-url'
+const { isImageHash, isLocalImg } = require('lib/boolean_validations')
+const { shouldNotBeCalled } = require('../utils/utils')
+
+const convertUrl = url => authReq('post', endpoint, { url })
+
+describe('images:convert-url', () => {
+  it('should convert a URL', async () => {
+    const imageUrl = 'https://raw.githubusercontent.com/inventaire/inventaire-client/master/app/assets/icon/32.png'
+    const { url, hash } = await convertUrl(imageUrl)
+    isImageHash(hash).should.be.true()
+    isLocalImg(url).should.be.true()
+  })
+
+  it('should reject an URL returning a 404', async () => {
```

Comment:
maybe a test on rejecting non URL string?

Reply:
done in 5dcfc4625